hil-test/tests/spi_full_duplex.rs

```diff
@@ -480,4 +488,89 @@ mod tests {
 
         assert_eq!(&[0xff, 0xff, 0xff, 0xff], dma_rx_buf.as_slice());
     }
+
+    #[test]
+    #[timeout(3)]
+    fn cancel_stops_transaction(mut ctx: Context) {
+        // Slow down
+        ctx.spi.change_bus_frequency(100.Hz());
```

Comment:
This might be a bit misleading, on the device I tested this (I forgot which one it was, probably ESP32), the lowest it could go was about 80kHz.

Reply:
Hardware limitation or bug in driver?

Reply:
Hardware limitation, this is as low as a 4-bit prescaler and a 6-bit divider gives us.